LootLockerServerSDK/LootLockerServerSDK.uplugin

```diff
@@ -2,7 +2,7 @@
     "FileVersion": 3,
     "FriendlyName": "LootLockerServerSDK",
     "Version": 4,
-    "VersionName": "3.0.1",
+    "VersionName": "3.0.2",
```

Comment:
Wouldn't this be a 3.1.0 since we're adding new features?

Reply:
You're correct, thought I had already done that >.<

Reply:
Fixed